src/services/moviedbService.js

```diff
@@ -1,7 +1,7 @@
 // TODO: make this key an envvar for the love of god
-const API_KEY = '6af1acbb5b00250f0669d50b891c76c6'
-const MOVIEDB_API_BASE_URI = 'https://api.themoviedb.org/3'
-const POSTER_API_BASE_URI = 'https://image.tmdb.org/t/p/w500'
+const API_KEY = process.env.REACT_APP_MOVIEDB_API_KEY;
```

Comment:
onde tá isso?

Reply:
no arquivo '.env' que não vai subir para o github. Por enquanto posso só te mandar ele, mas se mais gente for mexer talvez tenha que pensar em outra solução.

Reply:
ah sim kkkkkkkkkkkkkkkkkkkkk n pensei 🤦  my bad